signer/signer.go

```diff
@@ -139,6 +139,17 @@ type Configuration struct {
 	// CaCert is the certificate of the root of the pki, when used
 	CaCert string `json:"cacert,omitempty"`
 
+	// Hash is a hash algorithm like 'sha1' or 'sha256'
```

Comment:
nit: s/like//

Reply:
generic option, could contain md5 or sha512 in the future, though hopefully not...